run-test262.c

```diff
@@ -820,6 +820,19 @@ static JSModuleDef *js_module_loader_test(JSContext *ctx,
     uint8_t *buf;
     JSModuleDef *m;
     JSValue func_val;
+    char *filename, *slash, path[PATH_MAX];
+
+    // interpret import("bar.js") from path/to/foo.js as
+    // import("path/to/bar.js") but leave import("./bar.js") untouched
+    filename = opaque;
+    if (!strchr(module_name, '/')) {
+        slash = strrchr(filename, '/');
+        if (slash) {
+            snprintf(path, sizeof(path), "%.*s/%s",
+                     (int) (slash - filename), filename, module_name);
```

Comment:
Looks like an include is missing for PATH_MAX

Reply:
No, it's that not all platforms define PATH_MAX in that technically-right-but-there's-a-reason-you're-never-invited-to-parties sense.